src/css/profile/mobile/common/appbar.less

```diff
@@ -222,6 +226,7 @@
 	}
 
 	.ui-label-select-all {
+		.font(regular);
```

Comment:
Would you check the  3px padding between checkbox and "All" text,, please?

Reply:
Thank you.
It is the part I didn't check.
Done.